schema.sql

```diff
@@ -28,15 +28,17 @@ CREATE SCHEMA web3storage
     id BIGSERIAL PRIMARY KEY,
     user_id bigint NOT NULL,
     auth_key_id bigint,
-    content_id bigint NOT NULL,
+    content_cid TEXT NOT NULL,
+    -- CID for content as provided by the user
+    cid TEXT NOT NULL,
```

Comment:
I think it would be a lot more intuitive to rename `content_cid` to `cid` so it matches name in the `content` table and call this `source_cid`, `provided_cid` or `original_cid`, so it's clear without reaching for comments.

```suggestion
    source_cid TEXT NOT NULL,
```
prefixing both `cid`s is also more clear e.g. `content_cid` and `upload_cid`.

Reply:
Another alternative could be calling normalized `cid` as `cid_v1_b32` so it's self descriptive.

Reply:
I'm happy to go with `source_cid` - it has similar meaning in dagcargo.

`cid_v1_b32` is the normalized form we're using now. I'd rather not encode this as a column name.

In my experience foreign keys often are prefixed with the table they reference so you can know what it references without inspecting the schema. So I've used that convention here with `user_id`, `pin_location_id` etc. etc. So it seems consistent to me to call this `content_cid`. Note that convention seems to also be used in the [niftysave schema](https://github.com/ipfs-shipyard/nft.storage/pull/394).

Reply:
makes sense `source_cid` and `content_cid` ✅